GDevelop.js/Bindings/Bindings.idl

```diff
@@ -1933,6 +1938,21 @@ interface QuickCustomizationVisibilitiesContainer {
     QuickCustomization_Visibility Get([Const] DOMString name);
 };
 
+interface Screenshot {
+    [Const] long GetTiming();
+    void SetTiming([Const] long timing);
```

Comment:
No [Const] or [Ref] for primitive values.

```suggestion
    long GetTiming();
    void SetTiming(long timing);
```

Reply:
(Const is mostly useless for these, they are passed by value, i.e: copied in memory. Const is useful when you pass a reference, and you don't want it to be modified. Often the case for a string or an object)